src/main/msp/msp_serial.c

```diff
@@ -231,14 +316,22 @@ void mspSerialProcess(mspEvaluateNonMspData_e evaluateNonMspData, mspProcessComm
                 } else if (mspPort->packetType == MSP_PACKET_REPLY) {
                     mspSerialProcessReceivedReply(mspPort, mspProcessReplyFn);
                 }
+
+                mspPort->c_state = MSP_IDLE;
```

Comment:
this state change code was moved from mspSerialProcessReceivedCommand so that it is more obvious where the port state is updated.